src/app/Main.java

```diff
@@ -1,53 +1,59 @@
 package app;
 
+import data_access.InMemoryGameDataAccessObject;
 import interface_adapter.ViewManagerModel;
 import interface_adapter.game_over.GameOverViewModel;
 import interface_adapter.game_settings.GameSettingsViewModel;
 import interface_adapter.menu.MenuViewModel;
 import interface_adapter.round.RoundViewModel;
+import interface_adapter.submit_answer.SubmitAnswerViewModel;
 import view.*;
 
 import javax.swing.*;
 import java.awt.*;
 
 public class Main{
     public static void main(String[] args){
-
+        // Setup JFrame app
         JFrame application = new JFrame("Spotify Bandits");
         application.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
-
         CardLayout cardLayout = new CardLayout();
-
         application.setMinimumSize(new Dimension(300, 200));
-
         JPanel views = new JPanel(cardLayout);
         application.add(views);
 
+        // Setup view manager
         ViewManagerModel viewManagerModel = new ViewManagerModel();
         new ViewManager(views, cardLayout, viewManagerModel);
 
-        // Create Views
+        // Create view models
         MenuViewModel menuViewModel = new MenuViewModel();
-        MenuView menuView = new MenuView(menuViewModel, viewManagerModel);
-        views.add(menuView, menuView.viewName);
-
         GameSettingsViewModel gameSettingsViewModel = new GameSettingsViewModel();
-        GameSettingsView gameSettingsView = new GameSettingsView(gameSettingsViewModel, viewManagerModel);
-        views.add(gameSettingsView, gameSettingsView.viewName);
-
         GameOverViewModel gameOverViewModel = new GameOverViewModel();
+        RoundViewModel roundViewModel = new RoundViewModel();
+        SubmitAnswerViewModel submitAnswerViewModel = new SubmitAnswerViewModel();
+
+        // Create data access objects
+        InMemoryGameDataAccessObject gameDataAccessObject = new InMemoryGameDataAccessObject();
```

Comment:
I believe this is to get the game running since we don't have the presentient data set up yet and once we do set it up I believe we will need to update this correct?

Reply:
I'm not sure if I interpreted your question correctly, but I believe the answer is yes. We will have to inject some sort of game data access object into the various interactors that need one (e.g.`SubmitAnswerInteractor`) so creating a common one here to use for all of them makes sense I think. We will definitely need to swap out the in-memory DAO for one that persists beyond the current program execution though if that is what you're referencing. Let me know if this answered your question :)

Reply:
Yes exactly, we will need to replace the in memory DAO for persistent DAO once we have that created perfect!